Cython/Utility/TypeConversion.c

```diff
@@ -1,8 +1,12 @@
 /////////////// TypeConversions.proto ///////////////
-//@requires: StringTools.c::IncludeStringH
 
 /* Type Conversion Predeclarations */
 
+// Note that this is *always* included in the module preamble
+// (but the impl part is only generated on demand).
+// Therefore IncludeStringH is not be reliable
+#include <string.h>
```

Comment:
I've started writing a cleanup to at least avoid needing this in the `.proto` part. It should be enough to include it in the impl section. We can then exclude these changes from this PR.

Reply:
OK. I'll drop this part